tools/ucalls.py

```diff
@@ -14,8 +14,20 @@
 
 from __future__ import print_function
 import argparse
-from bcc import BPF, USDT
+from bcc import BPF, USDT, USDTException
 from time import sleep
+import sys
+
+def enable_probe_or_bail(usdt, probe, function):
+    try:
+        usdt.enable_probe(probe, function)
+    except USDTException:
```

Comment:
It probably should, but I wasn't sure if polluting the module with this check was worth it. Because it's essentially USDT::enable_probe with a custom behavior on failure. What do you think?